rules/integrations/aws/defense_evasion_s3_bucket_lifecycle_expiration_added.toml

```diff
@@ -0,0 +1,109 @@
+[metadata]
+creation_date = "2024/04/12"
+integration = ["aws"]
+maturity = "production"
+min_stack_comments = "AWS integration breaking changes, bumping version to ^2.0.0"
+min_stack_version = "8.9.0"
+updated_date = "2024/05/09"
+
+[rule]
+author = ["Elastic"]
+description = """
+Identifies an expiration lifecycle configuration added to an S3 bucket. Lifecycle configurations can be used to manage
+objects in a bucket, including setting expiration policies. This rule detects when a lifecycle configuration is added to
+an S3 bucket, which could indicate that objects in the bucket will be automatically deleted after a specified period of
+time. This could be used to evade detection by deleting objects that contain evidence of malicious activity.
+"""
+false_positives = [
```

Comment:
will it make sense to use a new term rule type for this use case ?

Reply:
IMO, I think we alert any time an expiration is given to inform admins. Hiding footprints via log deletion is very common in cloud attacks. Cloudtrail logs for auditing are pushed into S3 buckets which are then sent into the SQS queue to be pulled by the agent. If a bucket is given an expiration and is then deleted, that could tank visibility and threat detection all together.